app/src/types/pages/feedbackPage/types.ts

```diff
@@ -0,0 +1,23 @@
+export type FormData = Record<FORM_FIELDS, string>;
+
+export enum FORM_FIELDS {
+    feedbackContent = 'feedbackContent',
+    howSatisfied = 'howSatisfied',
+    respondentName = 'respondentName',
+    respondentEmail = 'respondentEmail',
+}
+
+export enum SUBMISSION_STAGE {
+    NotSubmitted,
+    Submitting,
+    Successful,
+    Failure,
+}
+
+export enum SATISFACTION_CHOICES {
+    VerySatisfied = 'Very satisfied',
+    Satisfied = 'Satisfied',
+    Neither = 'Neither satisfied or dissatisfied',
+    Dissatisfied = 'Dissatisfied',
+    VeryDissatisfied = 'Very dissatisfied',
```

Comment:
Not sure if we should be using enums as strings to render. These are reserved for logical choices within the code. If we were to apply a language library such as i18n we can't use it here. Please extract this into a regular key:value object if you prefer this method for label:value

Reply:
Thanks. Changed SATISFACTION_CHOICES to a plain key:value object